dist/src/main/config/broker.yaml.template

```diff
@@ -201,6 +201,100 @@
       # This setting can also be overridden using the environment variable ZEEBE_BROKER_THREADS_IOTHREADCOUNT
       # ioThreadCount: 2
 
+    # backpressure:
+      # Configure backpressure below
+      #
+      # Set this to enable or disable backpresure.
+      # This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_ENABLED
+      # enabled : true
+
+      # if enabled - will use the average latencies over a window as the current latency to update the limit.
+      # It is not recommended to enable this when the algorithm is aimd. This setting is not applicable to fixed limit.
+      # This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_USEWINDOWED
+      # useWindowed: true
+
+      # The algorithm configures which algorithm to use. It should be one of vegas, aimd, fixedLimit, gradient, or gradient2.
+      # You then only need to configure the subsection relating to the algorithm of your choice.
+      # This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_ALGORITHM
+      # algorithm: "vegas"
+
+      # Configure the following if the algorithm is "aimd".
+      # AIMD increases the limit for every successfully response and decrease the limit for every request timeout.
+      # aimd:
```

Comment:
Do we want to say something about back pressure and algorithms in the documentation?

Reply:
There is already some documentation about backpressure. But there is no documentation about the algorithms. I will create a follow up issue to add docs. Is that ok?

Reply:
sure